core-tests/shared/src/test/scala/zio/ZIOSpec.scala

```diff
@@ -827,18 +827,19 @@ object ZIOSpec extends ZIOBaseSpec {
         assertM(ZIO.forkAll(list.map(a => ZIO.effectTotal(a))).flatMap(_.join))(equalTo(list))
       },
       testM("empty input") {
-        assertM(ZIO.forkAll(List.empty).flatMap(_.join))(equalTo(List.empty))
+        assertM(ZIO.forkAll(List.empty[ZIO[Any, Nothing, Unit]]).flatMap(_.join))(equalTo(List.empty))
```

Comment:
This is the price we pay, eh?

Reply:
It works fine on Scala 2.12. We don't have to pay it if https://github.com/scala/bug/issues/12104 is resolved.